frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.test.tsx

```diff
@@ -1,29 +1,128 @@
 import React from 'react';
-import { ClassificationForm } from './ClassificationForm';
 import renderer from 'react-test-renderer';
-import { Formik } from 'formik';
-import { Classifications } from 'constants/classifications';
 import { cleanup, fireEvent, render, wait } from '@testing-library/react';
+import { Formik } from 'formik';
 import { noop } from 'lodash';
+import axios from 'axios';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
```

Comment:
If I get some time I'd like to write a test component that provides a wrapper for the router/memory router and the provider/store, since some tests don't really care about setting that stuff up but they have to because of a useRouter or redux call somewhere.

Reply:
Oooh that would be handy